app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
```

Comment:
Protect that controller. 
Only connected user must be able to have access to its actions
Use `before_action` to do that